backend/graph/nodes.py

```
@@ -0,0 +1,57 @@
+    summary = state.summary
+
+    if summary:
+        summary_message = (
+            f"This is summary of the conversation to date: {summary}\n\n"
+            "Extend the summary by taking into account the new messages above:"
+        )
+
+    else:
+        summary_message = "Create a summary of the conversation above:"
+
+    messages = state.messages + [HumanMessage(content=summary_message)]
```

Ruff (0.15.0)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)